src/utils/environment.ts

```diff
@@ -0,0 +1,4 @@
+export const getApiUrl = (environment: string) =>
+  environment === 'dev'
+    ? 'https://api.dev.dreader.io'
+    : 'https://api.dreader.io';
```

Comment:
I'll move this value to .env, and also gonna add the webhook ID

Reply:
We can use the `DOMAIN_NAME` env value for the domain, and `WEBHOOK_ID` for for the webhook.
In the future we will keep track of webhooks in a database table potentially